src/Documentation/sidebar.json

```diff
@@ -186,15 +186,16 @@
     }
   },
   {
-    "name": "DVC Philosophy",
+    "name": "Understanding DVC",
```

Comment:
I would rename the directory itself as well. The one that contains all these files.

Reply:
changed the folder name as well :)